pkg/pod/pods.go

```diff
@@ -1187,3 +1189,29 @@ func (p *Pod) AppExitCode(appName string) (int, error) {
 	statusFile := filepath.Join(stage1RootfsPath, "/rkt/status/", appName)
 	return p.readIntFromFile(statusFile)
 }
+
+// ManifestExclusiveLock gets an exclusive lock on only the pod manifest.
+// This is used in the app sandbox - since the pod is already running, we
+// won't be able to get an exclusive lock on the pod itself.
+func (p *Pod) ExclusiveManifestLock() error {
+	l, err := lock.ExclusiveLock(common.PodManifestLockPath(p.Path()), lock.RegFile)
```

Comment:
No, I'm mirroring the locking semantics used throughout `rkt`: locking/unlocking is idempotent. I don't see a reason to differ in this case alone.